ui/src/index.tsx

```diff
@@ -110,7 +111,36 @@ const parsePodStatus = (pod: any) => {
   }
 };
 
-const parseReplicaSets = (tree: any, rollout: any) => {
+const parseAnalysisRuns = (tree: any, rollout: any): RolloutAnalysisRunInfo[] => tree.nodes
+    .filter(node => (node.kind === 'AnalysisRun') && (node.parentRefs.some(ref => ref.name === rollout.metadata.name)))
+    .map(node => ({
+      objectMeta: {
+        creationTimestamp: {
+          seconds: node.createdAt,
+        },
+        name: node.name,
+        namespace: node.namespace,
+        resourceVersion: node.version,
+        uid: node.uid,
+      },
+      revision: parseRevision(node),
+      status: parseAnalysisRunStatus(node.health.status),
+    }) as RolloutAnalysisRunInfo);
+
+const parseAnalysisRunStatus = (status: string): string => {
+  switch(status) {
+    case 'Healthy':
+      return 'Successful';
+    case 'Progressing':
+      return 'Running';
+    case 'Degraded':
+      return 'Failure';
+    default:
+      return 'Error';
+  }
+}
```

Comment:
`parseAnalysisRunStatus` function converts Argo CD status to Argo Rollouts' Analysis Run status.